app/src/main/res/values/strings.xml

```diff
@@ -296,4 +296,5 @@
   <!-- WalkthroughActivity -->
   <string name="walkthrough_welcome_description">Learn new math skills with stories that show you how to use them in your daily life</string>
   <string name="welcome">"Welcome %s!"</string>
+  <string name="topic_name_text_view">Ratios and Proportional Reasoning</string>
```

Comment:
Remove this string as it is not getting used in production code. It is just getting used in test code and it is completely fine if we hard code this in test code. This is how it works:
If you are writing a test case in which you have a string to compare:
1. Check if the string already exists in `strings.xml`, if yes then use that resource in test code.
2. If not, then hard-code it in test case but do not introduce it in `strings.xml` file.

`strings.xml` will contain only those strings which are getting used in production code.

Reply:
Done